demo/repl.html

```diff
@@ -46,6 +46,10 @@
 <script src="../third_party/CodeMirror-4.0.3/keymap/sublime.js"></script>
 
 <script src="../bin/traceur.js"></script>
+<script>
+window.System = new traceur.runtime.BrowserTraceurLoader();
+System.map = System.semverMap(System.version);
```

Comment:
Just a hack in two ways:
  The new Loader spec has a different name for it.
  Creating names from version isn't a thing.
But it's not horrible. The spec will probably be close and the version
thing is all in "user space" not something we impose on users of traceur.

I'm trying to get around to updating to the new spec proposal....

On Tue, Mar 31, 2015 at 3:10 PM, Erik Arvidsson <notifications@github.com>
wrote:

> In demo/repl.html
> <https://github.com/google/traceur-compiler/pull/1855#discussion_r27528418>
> :
>
> > @@ -46,6 +46,10 @@
> >  <script src="../third_party/CodeMirror-4.0.3/keymap/sublime.js"></script>
> >
> >  <script src="../bin/traceur.js"></script>
> > +<script>
> > +window.System = new traceur.runtime.BrowserTraceurLoader();
> > +System.map = System.semverMap(System.version);
>
> What is the story about System.map? Is this the intended usage or are we
> just hacking this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/traceur-compiler/pull/1855/files#r27528418>.
>